<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewWidget</class>
 <widget class="QWidget" name="ViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Manhattan::StyledBar" name="toolBar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="createButton">
        <property name="text">
         <string>Create</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+1</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="transformButton">
        <property name="text">
         <string>Transform</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+2</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="parentEditButton">
        <property name="text">
         <string>Parent edit</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+3</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Manhattan::StyledSeparator" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QToolButton" name="bonesButton">
        <property name="text">
         <string>Bones</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="attachmentsButton">
        <property name="text">
         <string>Attachments</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Manhattan::StyledSeparator" name="widget_2" native="true"/>
      </item>
      <item>
       <widget class="QToolButton" name="viewParentalLinesButton">
        <property name="text">
         <string>View parental lines</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="View" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header>View.hpp</header>
   <slots>
    <signal>switchedToBoneTargetMode()</signal>
    <signal>switchedToAttachmentTargetMode()</signal>
    <slot>setTransformEditMode()</slot>
    <slot>setRotateTransformMode()</slot>
    <slot>setTranslateTransformMode()</slot>
    <slot>setScaleTransformMode()</slot>
    <slot>setCreateEditMode()</slot>
    <slot>setSelectTransformMode()</slot>
    <slot>setParentEditMode()</slot>
    <slot>setBoneTargetMode()</slot>
    <slot>setAttachmentTargetMode()</slot>
    <slot>setParentalLinesVisible(bool)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Manhattan::StyledBar</class>
   <extends>QWidget</extends>
   <header location="global">styledbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Manhattan::StyledSeparator</class>
   <extends>QWidget</extends>
   <header location="global">styledbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>createButton</sender>
   <signal>pressed()</signal>
   <receiver>graphicsView</receiver>
   <slot>setCreateEditMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transformButton</sender>
   <signal>pressed()</signal>
   <receiver>graphicsView</receiver>
   <slot>setTransformEditMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parentEditButton</sender>
   <signal>pressed()</signal>
   <receiver>graphicsView</receiver>
   <slot>setParentEditMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>attachmentsButton</sender>
   <signal>pressed()</signal>
   <receiver>graphicsView</receiver>
   <slot>setAttachmentTargetMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bonesButton</sender>
   <signal>pressed()</signal>
   <receiver>graphicsView</receiver>
   <slot>setBoneTargetMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphicsView</sender>
   <signal>switchedToAttachmentTargetMode()</signal>
   <receiver>attachmentsButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphicsView</sender>
   <signal>switchedToBoneTargetMode()</signal>
   <receiver>bonesButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewParentalLinesButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>setParentalLinesVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
